elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/client/impl/BackendRequestProcessor.java

```diff
@@ -224,6 +239,11 @@ public void shutdown() {
 			scheduler.shutdown();
 			try {
 				scheduler.awaitTermination( Long.MAX_VALUE, TimeUnit.SECONDS );
+				if ( lastAsyncWorkLatch != null ) {
+					//It's possible that a task was successfully scheduled but had no chance to run,
```

Comment:
Actually it would be nice to not skip this block on interrupted. I should move it into a finally after the catch.